sdk/azcore/tracing/tracing.go

```diff
@@ -19,16 +19,18 @@ type ProviderOptions struct {
 // NewProvider creates a new Provider with the specified values.
 //   - newTracerFn is the underlying implementation for creating Tracer instances
 //   - options contains optional values; pass nil to accept the default value
-func NewProvider(newTracerFn func(name, version string) Tracer, options *ProviderOptions) Provider {
+func NewProvider(newTracerFn func(name, version string) Tracer, newPropagatorFn func() Propagator, options *ProviderOptions) Provider {
```

Comment:
This is a breaking change and is what CI is complaining about. You'll need to put `newPropagatorFn` in `ProviderOptions`.